docs/guides/components_v2/advanced.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: Guides.ComponentsV2.Advanced
+title: Create Components V2
+---
+
+# Component types
+
+As denoted in [Intro], This framework supports a lot of component types. This example offers some more insight into what your options are. Below is a component with `TextDisplay`, `MediaGallery` and `ActionRow` (with `Buttons` or `SelectMenu`).
+
+![](images/interaction-response.png)
```

Comment:
This should have a full list on components, which one can be nested adh which childs they accept + limits.
i.e. `SectionComponent` can have up to 3 `TextDisplay` components + 1 Accessory, either a thumbnail or a button